Docs: adds SAML and Okta to What's new in 7.0 (#24629)
grafana/grafana: docs/sources/guides/whats-new-in-v7-0.md

The main highlights are:

- [**New Panel Editor** Redesign based on community feedback.]({{< relref "#new-panel-editor" >}})
- [**Explore** New tracing UI and support for visualizing Jaeger and Zipkin traces.]({{< relref "#new-tracing-ui" >}})

> - [**Enterprise** Usage Analytics, Presence and Auth Improvements.]({{< relref "#grafana-enterprise" >}})

Reviewer: ```suggestion
- [**Enterprise** Usage Insights, Presence Indicator and Auth Improvements.]({{< relref "#grafana-enterprise" >}})
```
It would be nice to update this in the subheader as well (https://github.com/grafana/grafana/blob/master/docs/sources/guides/whats-new-in-v7-0.md#usage-analytics-and-presence)

Reviewer 2: And apply the serial comma after "Presence indicator".

Author: Good points, thanks!

{{< docs-imagebox img="/img/docs/v70/presence_indicator.jpg" max-width="1024px" caption="Grafana Enterprise - Presence Indicator" >}}

### SAML Role and Team Sync

> SAML support in Grafana Enterprise is improved by adding Role and Team Sync. Read more about how to use these features in the [documentation]({{< relref "../enterprise/saml.md#configure-team-sync" >}}).

Reviewer 3: For people using screen readers, it's always a good idea to make sure the link isn't too generically sounding.

```suggestion
SAML support in Grafana Enterprise is improved by adding Role and Team Sync. Read more about how to use these features in the [SAML team sync documentation]({{< relref "../enterprise/saml.md#configure-team-sync" >}}).
```

Author: Good point!

### Okta OAuth Team Sync

> Okta gets its own provider which adds support for Team Sync. Read more about using Okta in the [documentation]({{< relref "../auth/okta.md" >}}).

Reviewer 3: ```suggestion
Okta gets its own provider which adds support for Team Sync. Read the [Okta documentation]({{< relref "../auth/okta.md" >}}) to learn more about using the Okta provider.
```

possibly repeat that it's specifically for Okta w/ OAuth since Okta provides multiple ways to connect to third-party applications

```suggestion
Okta gets its own OAuth provider which adds support for Team Sync. Read the [Okta documentation]({{< relref "../auth/okta.md" >}}) to learn more about using the Okta provider.
```
